docs/debugging.md

```diff
@@ -0,0 +1,27 @@
+# Debugging
+
+In your project, if you find that there is a bug that appears ONLY when you use BabelMinify, it's most likely that there is a bug in BabelMinify and you should definitely report it. Here are some guidelines that might help you drill down the issue. If it doesn't help you, you can of course create a minimal repro project with the bug and report it.
```

Comment:
Can we extend this and add like
`when you use BabelMinify standalone or babel-minify as a preset in babelrc`